content/applications/productivity/calendar/outlook.rst

```diff
@@ -41,37 +70,72 @@ URI (URL) followed by `/microsoft_account/authentication`.
 For more information on the restrictions and limitations of URIs, `check this page <https://docs.
 microsoft.com/en-us/azure/active-directory/develop/reply-url>`_.
 
-Regarding the application credentials, the user *must* add a client secret, which allows Odoo to
-authenticate itself, requiring no interaction from the user's side. :guilabel:`Certificates` are
-optional.
+Finally, on the application registration page, click :guilabel:`Register` button to complete the
+application registration. The :guilabel:`Application (client) ID` is produced. Copy this value, as
+it is needed later, in the :ref:`outlook_calendar/odoo_setup`.
+
+.. image:: outlook/app-client-id.png
+   :align: center
+   :alt: Application client ID highlighted in the essentials section of the newly created
+         application.
+
+Create client secret
+--------------------
+
+The second credential needed to complete the synchronization of the Microsoft *Outlook Calendar* is
+the *Client Secret*. The user **must** add a client secret, as this allows Odoo to authenticate
+itself, requiring no interaction from the user's side. *Certificates* are optional.
+
+To add a client secret, click :guilabel:`Certificates & secrets` in the left menu. Then click
```

Comment:
When directing the user to select a menu item, it is preferred to use the menuselection tag :)
```suggestion
To add a client secret, click :menuselection:`Certificates & secrets` in the left menu. Then click
```